### ISSUE TITLE
Upon exit, user should be warned about ongoing normal swap

### ISSUE THREAD

**Issue by bitromortac** [spesmilo/electrum#7439]
Currently, there is no warning when a user closes Electrum but a normal swap is ongoing. If a user forgets about the swap, the swap server can never fulfill it.

In the normal swap case, the user needs to be online to receive the offchain transaction, which can take a while because the onchain lockup transaction takes some time to confirm. To mitigate this issue we could inform the user about the ongoing swap and ask for staying connected.

**Comment by SomberNight** [comment]
Good point. So the scenario is that the user sends an onchain transaction and the server is waiting for that tx to confirm before it pays via LN; and the user should stay online until the server pays.
(If the user goes offline, they will have to wait for the timelock on the onchain output to expire and reclaim their UTXO via another tx.)

**Comment by SomberNight** [comment]
I believe this was implemented in https://github.com/spesmilo/electrum/pull/9715.

Also note that normal swaps work differently now (than described above), since https://github.com/spesmilo/electrum/pull/8566.
See 
https://github.com/spesmilo/electrum/blob/61623d6d56282bc344d22bfa70d93c332964cd87/electrum/submarine_swaps.py#L713-L721
